NEWS.md

```diff
@@ -1,8 +1,28 @@
+# admiral.test 0.5.0
+ - Updated AE to add variable AELAT for ophthalmology package (#99)
+ - Updated TR, TU, RS for admiralonco package (#103):
+     - tumor location `"LYMPH NODE"` added
+     - TR test codes `"LDIAM"` and `"LPERP"` added
+     - unscheduled visits added
+     - incomplete assessments of target lesions added
+ - Updated QS to add questionnaire for ophthalmology package (#120)
+ - Created SDTM SC dataset for ophthalmology package (#102)
+ - Created SDTM OE dataset for ophthalmology package (#101)
+ - Using {metatools} to add labels to datasets (#87)
+ - `admiral.test` has SDTM data for [oncology](https://pharmaverse.github.io/admiralonco/main/index.html)
+ and [ophthalmology](https://pharmaverse.github.io/admiralophtha/main/reference/index.html)
+ - Removed `library(admiral.test)` from all programs in the `./dev` directory.
+ 
 # admiral.test 0.4.0
-
-- `get_smq_select()` and `get_sdq_select()` have been deprecated in favor of `get_terms()` to accommodate changes in {admiral} (#94)
- - Updated labels for PP and PC datasets (#77)
- - Implemented admiral CI/CD workflows
+ - Updated labels for pp and pc datasets (#77)
+ - Updates to use `get_terms` and `basket_select` (#94)
+ - Various updates to improve CI/CD workflows
```

Comment:
I am not sure why these lines were change. If we change them, we should use uppercase for the dataset names (PP and PC) and brackets should be added to the function names, i.e., `get_terms()` and `basket_select()` instead of `get_terms` and `basket_select`.

Reply:
I believe at some point the `NEWS.md` had changes made to `main` that somehow were not adopted into `devel` so we were actually missing these entries, and I agree it should actually just be changed back to the original text